cmd/jaeger/features/displayFeatures.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2023 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package features
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/confmap"
+	"go.opentelemetry.io/collector/confmap/provider/envprovider"
+	"go.opentelemetry.io/collector/confmap/provider/fileprovider"
+	"go.opentelemetry.io/collector/confmap/provider/httpprovider"
+	"go.opentelemetry.io/collector/confmap/provider/httpsprovider"
+	"go.opentelemetry.io/collector/confmap/provider/yamlprovider"
+	"go.opentelemetry.io/collector/otelcol"
+
+	"github.com/jaegertracing/jaeger/cmd/jaeger/internal"
+	"github.com/jaegertracing/jaeger/pkg/version"
+)
+
+func DisplayFeatures() {
+	info := component.BuildInfo{
+		Command:     "jaeger",
+		Description: internal.GetDescription(),
+		Version:     version.Get().GitVersion,
+	}
+
+	settings := otelcol.CollectorSettings{
+		BuildInfo: info,
+		Factories: internal.Components,
+		ConfigProviderSettings: otelcol.ConfigProviderSettings{
+			ResolverSettings: confmap.ResolverSettings{
+				ProviderFactories: []confmap.ProviderFactory{
+					envprovider.NewFactory(),
+					fileprovider.NewFactory(),
+					httpprovider.NewFactory(),
+					httpsprovider.NewFactory(),
+					yamlprovider.NewFactory(),
+				},
+			},
+		},
+	}
+
+	fmt.Println("\n" + internal.GetDescription())
+	fmt.Println("List of feature gate identifiers. Prefix with '-' to disable the feature. '+' or no prefix will enable the feature.\n ")
+	otelcol.FeatureCommand(settings)
```

Comment:
why does `otelcol.FeatureCommand` need any of the above? It seems completely irrelevant.

Suggestion: copy the core logic of FeatureCommand into Jaeger directly for now, so that we don't have to wait for OTEL Collector reease.